h/static/scripts/annotator/adder.js

```diff
@@ -102,9 +102,12 @@ function Adder(container, options) {
     // property rather than `display` so that we can compute its size in order to
     // position it before display.
     display: 'block',
-    position: 'absolute',
     visibility: 'hidden',
 
+    // take position out of flow and off screen initially
+    position: 'absolute',
+    top: '100%',
```

Comment:
Why the value `100%`? That will result in the adder being positioned at the Y position of the content height of the `<body>`, which I think could still result in extra vertical space being added to the page, if the content height extends all the way to the bottom of the page. I'd suggest using `top: 0` here so that we know it won't add any extra height to the document, unless the height of the body happens to be less than the height of the adder itself.